.claude/skills/allocations/scripts/sources.py

```
@@ -0,0 +1,74 @@
+ALLOC_GIT_ROOT = "allocation-benchmark/allocations"
+ALLOC_LOCAL_ROOT = "allocation-benchmark/build/allocations"
+
+
+class GitSource:
+    def __init__(self, commit):
+        self.commit = commit
+
+    def list_files(self, subdir):
+        git_dir = f"{ALLOC_GIT_ROOT}/{subdir}".rstrip("/")
+        try:
+            out = subprocess.check_output(
```

Ruff (0.15.5)

[error] `subprocess` call: check for execution of untrusted input

(S603)